spec/features/upload_descriptive_metadata_spec.rb

```diff
@@ -14,7 +14,7 @@
   end
 
   let(:csv) do
-    "title1:value,purl\nmy title,https://purl.stanford.edu/#{Druid.new(item.externalIdentifier).without_namespace}\n"
+    "druid,title1.value,purl\n#{Druid.new(item.externalIdentifier)},my title,https://purl.stanford.edu/#{Druid.new(item.externalIdentifier).without_namespace}\n"
```

Comment:
This is for a single item, so I don't know why we need to provide a druid.

Reply:
Oh right, good call.  Removing.